src/cider/exec/processor/DefaultBatchProcessor.cpp

```diff
@@ -46,7 +46,7 @@ DefaultBatchProcessor::DefaultBatchProcessor(const plan::SubstraitPlanPtr& plan,
       std::make_shared<generator::SubstraitToRelAlgExecutionUnit>(plan_->getPlan());
   RelAlgExecutionUnit ra_exe_unit = translator->createRelAlgExecutionUnit();
   jitlib::CompilationOptions co;
-  co.dump_ir = true;
+  // co.dump_ir = true;
```

Comment:
Remove this entirely?

Reply:
we sometime need this for debug

Reply:
Minor hint: Let's add some comments for debug purpose.